CHANGELOG.md

```diff
@@ -80,11 +59,25 @@ This release removes the `cwl` top-level artifact. Some nonfunctional references
 
 For more information, see the [Cromwell 79 release notes](https://github.com/broadinstitute/cromwell/releases/tag/79).
 
-### TES backend supports `disks` attribute
+### TES Improvments
+
+* Surface TES System Logs to Cromwell when TES backend returns task error status
+Tes system errors are are now reported in Cromwell execution logs when the TES backend returns a task error.
```

Comment:
This needs a period and TES rendered in all caps. Actually, we can probably just take the first sentence out here since we don't need a headline anymore.